spec/jobs/sync_case_comments_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../spec_helper'
+
+RSpec.describe Jobs::SyncCaseComments do
+  include_context "spec helper"
+
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:_case) { Fabricate(:salesforce_case, topic: topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+  fab!(:post2) { Fabricate(:post, topic: topic, post_number: 2) }
+  fab!(:post3) { Fabricate(:post, topic: topic, post_number: 3) }
+
+  it 'creates multiple case comment objects on Salesforce' do
```

Comment:
I feel like we need to test for all the different conditions in the job as well. For example, when the post already has a custom fields set etc...